install/kubernetes/helm/subcharts/galley/templates/settings.yaml.tpl

```diff
@@ -0,0 +1,254 @@
+{{ define "settings.yaml.tpl" }}
```

Comment:
Is this autogenerated from the default settings, or written by hand? Are we relying on PR reviews to maintain consistency between the two?